app/src/main/java/com/andrewKarachun0304/tmshomeworkandroid/HomeWork2Activity.kt

```diff
@@ -0,0 +1,22 @@
+package com.andrewKarachun0304.tmshomeworkandroid
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+
+class HomeWork2Activity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_home_work2)
+    }
```

Comment:
I'm not sure,but I think you still need to define setOnnClickListeners to buttons

https://stackoverflow.com/questions/25803727/android-setonclicklistener-method-how-does-it-work

Watch the second answer here